### PR TITLE
Make Notebook appearence of Tables more consistent

### PR DESCRIPTION
**Pull request by eteq**
This makes some subtle changes to how Tables appear in notebooks - the main user-facing change is that displaying a table in a notebook (i.e., via ``Table._repr_html_``) now shows the table with the same (in my opinion, much more readable) style as ``show_in_notebook``.  It also gives the user a way to change this default via a new configuration item (``astropy.table.default_notebook_table_class``).

Note that if it's judged that the configuration item isn't desired, I can easily roll that back, but I would still push for having the default ``Table._repr_html`` match ``show_in_notebook``, whatever it might be.

Additionally, it updates the Table getting started docs to reflect this new appearence, and *also* adds mention of `show_in_notebook` (which I was shocked to find isn't in getting started despite being something I use almost literally every day).

cc @taldcroft

### CODE REVIEW

**Comment by astrofrog** [comment]
@eteq - you'll need to rebase this due to a changelog conflict. Otherwise 👍  from me!

**Comment by taldcroft** [comment]
Err, :-1: on this unless you can make the table not default to taking 100% width.
image

**Comment by eteq** [comment]
@taldcroft - hmm, I saw that as a feature, not a bug.  What size do you want them to be instead of 100%?

**Comment by taldcroft** [comment]
@eteq - the table should be only as wide as required by the data.  In other words, apart from style details it should match the way a plain HTML table is rendered, with minimal whitespace.

**Comment by taldcroft** [comment]
E.g. [these tables](http://nbviewer.jupyter.org/urls/bitbucket.org/hrojas/learn-pandas/raw/master/lessons/04%20-%20Lesson.ipynb) would look pretty silly (and be much more difficult to visually parse) if they were 100% of the width.

**Comment by eteq** [comment]
Ok, gotcha.  How about the new version I just pushed up? 

image

**Comment by eteq** [comment]
(had to rebase)

File: astropy/table/table.py [hunk 1]
```diff
@@ -889,9 +890,12 @@ def show_in_notebook(self, tableid=None, css=None, display_length=50,
             multiple times.
         table_class : str or `None`
             A string with a list of HTML classes used to style the table.
-            Default is "table table-striped table-bordered table-condensed",
-            using Bootstrap which is available in the notebook. See `this page
-            <http://getbootstrap.com/css/#tables>`_ for the list of classes.
+            The special default string ('from-apy-default') means that the string
+            will be retreived from the configuration item
+            ``astropy.table.default_notebook_table_class``. Note that these
+            table classes may make use of bootrap, as this is looded with the
```
Comment (taldcroft) [inline astropy/table/table.py]: typos: `bootstrap` and `loaded`

**Comment by taldcroft** [comment]
Yay, thanks!  I didn't actually test this but I assume from your example and doc update that tables are always "condensed".   If so, :+1:, nice!

**Comment by eteq** [comment]
Yep, that's right - turns out bootstrap arbitrarily sets things with the ``table`` class to width=100% so just removing that does the job. 

Fixed the typos you noticed, so this should be good to go.

**Pull request merged**